### PR TITLE
At/cb247/minors 8

### PR DESCRIPTION
**Pull request by Angelbt12**
https://inube.atlassian.net/jira/software/projects/CB/boards/20?selectedIssue=CB-247

### CODE REVIEW

File: src/components/modals/ReciprocityModal/index.tsx [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 import { Stack } from "@inubekit/stack";
-import { inube } from "@inubekit/foundations";
 import { Text } from "@inubekit/text";
 import { Icon } from "@inubekit/icon";
 import { useMediaQuery } from "@inubekit/hooks";
 import { Divider } from "@inubekit/divider";
-import { Textfield } from "@inubekit/textfield";
 import { Button } from "@inubekit/button";
+
 import { currencyFormat } from "@src/utils/formatData/currency";
```
Comment (SelsaErojas) [inline src/components/modals/ReciprocityModal/index.tsx:11]: Adjust the import aliases using the alias for components and adjust order of the import.

File: src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx [hunk 2]
```diff
@@ -355,7 +355,7 @@ export const ComercialManagement = (props: ComercialManagementProps) => {
             portalId="portal"
```
Comment (SelsaErojas) [inline src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx:355]: Adjust the import aliases using the alias for components and adjust order of the import.

**Review by SelsaErojas — CHANGES_REQUESTED** [review]

File: src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx [hunk 3]
```diff
@@ -324,7 +324,7 @@ export const ComercialManagement = (props: ComercialManagementProps) => {
           <CreditLimit
             handleClose={handleCloseModal}
             title="Origen de cupo"
-            portalId="portal"
```
Comment (SelsaErojas) [inline src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx:327]: ¿why change name of portal?
image

**Review by SelsaErojas — CHANGES_REQUESTED** [review]

**Review by SelsaErojas — APPROVED** [review]
Functional

File: src/components/modals/CreditLimit/index.tsx [hunk 4]
```diff
@@ -212,7 +217,8 @@ export const CreditLimit = (props: ICreditLimitProps) => {
               spacing="narrow"
             />
             <Text margin="0px 5px" size="small">
-              {creditLimitTexts.maxUsableLimitNote}
+              El menor de los anteriores es su cupo <strong>máximo</strong>{" "}
```
Comment (carin91) [inline src/components/modals/CreditLimit/index.tsx:220]: Relocate it into a config file...
I would use a jsx script to achieve this.

File: src/components/modals/PaymentCapacityModal/interface.tsx [hunk 5]
```diff
@@ -203,25 +201,27 @@ export const PaymentCapacityInterface = forwardRef<
               spacing="wide"
             />
             <Text margin="0px 5px" size="small" type="body">
-              {incomeModalConfig.infoText}
+              Monto máximo calculado para una cuota de
```
Comment (carin91) [inline src/components/modals/PaymentCapacityModal/interface.tsx:204]: Relocate this into the config file

**Review by carin91 — APPROVED** [review]

**Pull request merged**